### ISSUE TITLE
Wercker reports building rpm error

### ISSUE THREAD

**Issue by cosmo0920** [project-hatohol/hatohol#1496]
```log
Provides: libhatohol.so.0()(64bit)
Requires(interp): /bin/sh /bin/sh /bin/sh
Requires(rpmlib): rpmlib(CompressedFileNames) <= 3.0.4-1 rpmlib(FileDigests) <= 4.6.0-1 rpmlib(PayloadFilesHavePrefix) <= 4.0-1
Requires(pre): /bin/sh
Requires(post): /bin/sh
Requires(postun): /bin/sh
Requires: /bin/sh /usr/bin/env ld-linux-x86-64.so.2()(64bit) ld-linux-x86-64.so.2(GLIBC_2.3)(64bit) libc.so.6()(64bit) libc.so.6(GLIBC_2.2.5)(64bit) libc.so.6(GLIBC_2.3.4)(64bit) libc.so.6(GLIBC_2.4)(64bit) libcrypt.so.1()(64bit) libcrypto.so.10()(64bit) libdl.so.2()(64bit) libdl.so.2(GLIBC_2.2.5)(64bit) libgcc_s.so.1()(64bit) libgcc_s.so.1(GCC_3.0)(64bit) libgio-2.0.so.0()(64bit) libglib-2.0.so.0()(64bit) libgmodule-2.0.so.0()(64bit) libgobject-2.0.so.0()(64bit) libgthread-2.0.so.0()(64bit) libhatohol-common.so.0()(64bit) libhatohol.so.0()(64bit) libjson-glib-1.0.so.0()(64bit) libm.so.6()(64bit) libm.so.6(GLIBC_2.2.5)(64bit) libmlpl.so.0()(64bit) libmysqlclient.so.16()(64bit) libmysqlclient.so.16(libmysqlclient_16)(64bit) libnsl.so.1()(64bit) libpthread.so.0()(64bit) libpthread.so.0(GLIBC_2.2.5)(64bit) librabbitmq.so.1()(64bit) librt.so.1()(64bit) librt.so.1(GLIBC_2.2.5)(64bit) libsoup-2.4.so.1()(64bit) libsqlite3.so.0()(64bit) libssl.so.10()(64bit) libstdc++.so.6()(64bit) libstdc++.so.6(CXXABI_1.3)(64bit) libstdc++.so.6(GLIBCXX_3.4)(64bit) libstdc++.so.6(GLIBCXX_3.4.9)(64bit) libuuid.so.1()(64bit) libuuid.so.1(UUID_1.0)(64bit) libz.so.1()(64bit) python(abi) = 2.6 rtld(GNU_HASH)
Processing files: hatohol-arm-zabbix-15.06-1.el6.x86_64
error: File not found: /root/rpmbuild/BUILDROOT/hatohol-15.06-1.el6.x86_64/usr/sbin/hatohol-arm-plugin-zabbix
File not found: /root/rpmbuild/BUILDROOT/hatohol-15.06-1.el6.x86_64/usr/sbin/hatohol-arm-plugin-zabbix


RPM build errors:
2015/07/27 08:42:36 The command [/bin/sh -c cd ~/hatohol && scl enable devtoolset-2 "MAKEFLAGS=\"-j `cat /proc/cpuinfo | grep processor | wc -l`\" rpmbuild -tb hatohol-*.tar.bz2"] returned a non-zero code: 1
```

In more detail, https://app.wercker.com/#buildstep/55b5ec235604d5c21c13f940

**Comment by ashie** [comment]
Fixed in #1498

**Comment by cosmo0920** [comment]
Thanks! :yum:
